lib_device_control/host/device_access_i2c_rpi.c

```diff
@@ -90,7 +91,7 @@ control_read_command(control_resid_t resid, control_cmd_t cmd,
   unsigned char read_hdr[I2C_TRANSACTION_MAX_BYTES];
   unsigned len = control_build_i2c_data(read_hdr, resid, cmd, payload, payload_len);
   if (len != 3){
-    fprintf(stderr, "Error building read command section of read_device. len should be 3 but is %d\n", len);
+    PRINT_ERROR("Error building read command section of read_device. len should be 3 but is %d\n", len);
```

Comment:
s "error" would otherwise be repeated twice, I would substitute "failed to build for "Error building".